starboard/shared/uwp/extended_resources_manager.cc

```diff
@@ -439,6 +493,16 @@ void ExtendedResourcesManager::ReleaseExtendedResourcesInternal() {
 #endif
       d3d12device_.Reset();
     }
+    if (d3d12FrameBuffersHeap_) {
+#if !defined(COBALT_BUILD_TYPE_GOLD)
+      d3d12FrameBuffersHeap_->AddRef();
+      ULONG reference_count = d3d12FrameBuffersHeap_->Release();
+      SB_DLOG(INFO) << "Reference count of |d3d12device_| is "
```

Comment:
Change to SB_LOG so it will log in QA builds

Reply:
Done